configure.in

```diff
@@ -65,6 +66,10 @@ AS_IF([test -z "$krb5config"],
 	[ krb5config="$PATH_KRB5_CONFIG"]
 ])
 
+if test x$GCC = xyes ; then
+  WARNFLAGS="-Wall -Wextra -Wno-write-strings"
+fi
```

Comment:
Do we really want to run without any `WARNFLAGS` at all for non-GCC compilers? At least -Wall should be understood universally, shouldn't it?